samples/ChangeFeedMigrationTool/packages.config

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Azure.DocumentDB" version="1.14.1" targetFramework="net47" />
-  <package id="Microsoft.Azure.DocumentDB.ChangeFeedProcessor" version="1.0.0" targetFramework="net47" />
+  <package id="Microsoft.Azure.DocumentDB" version="1.17.0" targetFramework="net47" />
```

Comment:
@rnagpal Not sure, the Migration Tool project was made by @heyyjudes , probably with VS2017 that defaulted to 4.7 because the environment had that version?